manage_proj_page.php

```diff
@@ -255,9 +256,15 @@
 <?php
 			foreach( $t_categories as $t_category ) {
 				$t_id = $t_category['id'];
+				if ( $t_category['status'] == OFF){
+					$cat_status = lang_get( 'category_active' ) ;
```

Comment:
Should be `$t_category_status` (same name like used in manage_proj_edit_page.php)

Reply:
I have addressed all your comments in my latest commit.

Reply:
@CasN please test your changes before pushing. There are obvious syntax errors.